src/workloads/query/TelemetryQueryShapes.yml

```diff
@@ -0,0 +1,999 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests performance of telemetry with different query shapes.
+
+KeyWords:
+- telemetry
+- query
+- shape
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Repeat: &Repeat 10
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 6
+  nop: &Nop {Nop: true}
+  dbname: &db test
+
+Actors:
+# Clear any pre-existing collection state.
+- Name: ClearCollection
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: *Threads
+        Collection: *Collection
+        Operations:
+        - OperationName: drop
+
+- Name: InsertData
+  Type: Loader
+  Threads: 4
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          a: &integer {^RandomInt: {min: -100, max: 100}}
+          b: *integer
+          string: &string {^RandomString: {length: 5}}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+# Different Query Shapes.
+- Name: SimpleFind
+  Type: CrudActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {a: 1}
```

Comment:
Can you leave a comment here - we expect this to find ... 1e6 / 200 = 5000 documents? Or 0.5% of the collection?

Reply:
Oh sorry - outdated comment but that was from the [line 53 above](https://github.com/mongodb/genny/pull/862/files/ddbb03b58a243493317b4cbeb9af13e37ab63141#diff-b92ce3540ab5fab9fec3758b7cbcdcf64578ffdc7afd232fb8234cf5599e0e83R53) which defined
```yml
a: &integer {^RandomInt: {min: -100, max: 100}}
```
I guess maybe it is actually either 199 if those bounds are exclusive or 201 if they are inclusive, since 0 would sit right in the middle.